docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Let's use a constant color scale so we can compare the learned weights. Try</p><pre class="ql-syntax" spellcheck="false">plt.clim(-np.max(np.abs(W)), np.max(np.abs(W)))
</pre><p>so the color scale is centered on zero and all values have different colors.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>that doesn't work really well for some reason</p>